room.h

```diff
@@ -174,6 +174,15 @@ namespace QMatrixClient
             /** Mark all messages in the room as read */
             void markAllMessagesAsRead();
 
+            void inviteCall(const QString& callId, const int& lifetime,
+                            const QString& sdp);
+            void callCandidates(const QString& callId,
+                                const QJsonArray& candidates);
+            void answerCall(const QString& callId, const int& lifetime,
```

Comment:
`const int&`? Plain `int` will be more than enough :) The whole const-ref thing is a bit extraneous in case of Qt strings but we use it because it's more universal and also it still saves on refcounting. For `int` this just doesn't apply.

Reply:
Yeah, i'm getting to used to references :P